databrowser/src/domain/datasets/DatasetHero.vue

```diff
@@ -1,21 +1,22 @@
 <template>
   <HeroContainer>
-    <HeroTitle>All POIs</HeroTitle>
-    <HeroSubTitle
-      >Lorem ipsum dolor sit amen dolr sit amen dolr ist ist amen.
-    </HeroSubTitle>
+    <HeroTitle>{{ title }}</HeroTitle>
+    <HeroSubTitle>{{ subtitle }}</HeroSubTitle>
     <InternalLink to="/" tone="primary">More information</InternalLink>
   </HeroContainer>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
+import { defineProps, toRefs } from 'vue';
 import HeroContainer from '../../components/hero/HeroContainer.vue';
 import HeroTitle from '../../components/hero/HeroTitle.vue';
 import HeroSubTitle from '../../components/hero/HeroSubTitle.vue';
 import InternalLink from '../../components/link/InternalLink.vue';
-import { defineComponent } from '@vue/runtime-core';
 
-export default defineComponent({
-  components: { HeroContainer, InternalLink, HeroSubTitle, HeroTitle },
-});
+const props = defineProps<{
+  title?: string;
+  subtitle?: string;
+}>();
+
+const { title, subtitle } = toRefs(props);
```

Comment:
Uh, I didn't know that this is also working! I really like it. But should we dicuss if we should stick to this style in all components or switch to the other setup method that we used until now?

Reply:
I was also not aware of that until recently and I have to say that I find this style very pleasant to work with, because it forces you to use the composition API and think therefore accordingly. I agree that we should discuss if we move to this style.

Reply:
I saw also, that the handling of the component imports is way nicer. I like it a lot. We should really dicuss if we switch to this for all components.